src/features/events/tests/groupedEventsCard.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { GroupedEventsCards } from '../index'
+import { mockNodes, testEvent } from './__mocks__'
+import 'jest-styled-components'
+
+describe('The GroupedEventsCards component', () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)